mewarpx/mewarpx/util.py

```diff
@@ -5,9 +5,25 @@
 import inspect
 import os
 
+import numpy as np
+
 from pywarpx import geometry
 from mewarpx import mwxconstants as constants
 
+# Pass comm_world to sim.step(<steps>, <comm_world>) to send comm_world to amrex
```

Comment:
I think this actually will need to be passed to `mwxrun.init_run()` as that calls `self.simulation.initialize_warpx()`? Please add an option to pass this there if so (consider making it the default, actually, that can be overridden if desired?).